inference-engine/src/legacy_api/src/ie_cnn_layer_builder_ngraph.cpp

```diff
@@ -423,7 +423,7 @@ CNNLayer::Ptr NodeConverter<ngraph::op::Convert>::createLayer(const std::shared_
 
 template <>
 CNNLayer::Ptr NodeConverter<ngraph::op::Ceiling>::createLayer(const std::shared_ptr<ngraph::Node>& layer) const {
-    LayerParams params = {layer->get_friendly_name(), "Ceiling",
+    LayerParams params = {layer->get_friendly_name(), "Ceil",
```

Comment:
Does this renaming satisfy all plugins? Or it was just a mistake?

Reply:
Looks like it was WA to make this test work on GPU. So now these changes are reverted, but the Ceiling test on CPU fails for some reason, so it's commented out. @dmitry-gorokhov  FYI